pkg/model/message.go

```diff
@@ -12,5 +19,38 @@ type Message struct {
 	PluralID    string
 	Message     string
 	Description string
+	Positions   Positions
 	Fuzzy       bool
 }
+
+type Positions []string
+
+// Value implements driver.Valuer interface.
+func (p Positions) Value() (driver.Value, error) {
+	switch {
+	case p == nil:
+		return nil, nil
+	case len(p) == 0:
+		return "", nil
+	default:
+		return strings.Join(p, ", "), nil
+	}
+}
+
+// Scan implements sql.Scanner interface.
+func (p *Positions) Scan(value interface{}) error {
+	var positions sql.NullString
+	if err := positions.Scan(value); err != nil {
+		return fmt.Errorf("scan positions: %w", err)
+	}
+
+	if positions.Valid {
+		if len(positions.String) > 0 {
+			*p = strings.Split(positions.String, ", ")
+		} else {
+			*p = Positions{}
+		}
+	}
```

Comment:
why empty positions and not just nil ?
```suggestion
	if positions.Valid {
		if len(positions.String) > 0 {
			*p = strings.Split(positions.String, ", ")
		}
	}
```

Reply:
Tests will fail when persisting empty slice and then getting back nil from repo. The initial value will be compared to the returned one and they won't match.